ui/src/alerting/components/builder/HelpButton.tsx

```diff
@@ -5,7 +5,12 @@ import React, {FunctionComponent} from 'react'
 import {Button} from '@influxdata/clockface'
 
 const HelpButton: FunctionComponent = () => {
-  const handleClick = () => {}
+  const handleClick = () => {
+    const newTab = window.open(
+      'https://github.com/influxdata/docs-v2/blob/monitor-alert/content/v2.0/monitor-alert/checks/create.md#configure-the-check'
+    )
```

Comment:
That's just the markdown github link--live link will be https://v2.docs.influxdata.com/v2.0/monitor-alert/checks/create/#configure-the-check